### DISCUSSION TITLE
thanhle.blog/mot-cong-viec-te

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#84] [Ideas]
# thanhle.blog/mot-cong-viec-te

Những thứ mình thấy tệ nếu mình đặt trong hoàn cảnh làm việc như vậy

https://thanhle.blog/blog/mot-cong-viec-te

**Comment by khiem20tc** [comment]
Vậy anh có tin là sẽ có dream job mang lại cả 3 thứ tiền, kiến thức và giá trị bản thân kh anh? =))

**Reply by thanhlmm** [reply]
có, nhưng nó sẽ ko kéo dài và anh tin khoảng thời gian đó khá là ngắn ~ 1 năm

**Reply by khiem20tc** [reply]
em lại nghĩ chỉ có bản thân mình mới tự tạo ra công việc mang lại cả 3 thứ đó :)))

**Reply by thanhlmm** [reply]
haha. Thử đi em :3

**Comment by QuangNguyen1607** [comment]
Vô tình vào xem anh có ra bài nào không thì gặp đúng bài nói về vấn đề hiện tại của mình 😂.  Ngờ vực với những giá trị mình tạo ra, ngờ vực sao input thì nhiều mà output thì lại không như mình expected. Nhưng thôi vẫn cứ keep going :)))) rồi chắc chắn output sẽ như mình mong đợi :3 :3.

**Reply by thanhlmm** [reply]
haha, in the end, you always earn something 🔥

**Comment by hai123vn** [comment]
hi anh chúc anh 1 ngày tốt lành. Khi nào ra blog mới nữa anh nhỉ 🤣

**Reply by thanhlmm** [reply]
tuần sau nha :)))

**Reply by hai123vn** [reply]
đã qua tuần rồi anh nhỉ, em vẫn đang hóng đây anh ạ =))

**Reply by thanhlmm** [reply]
:)))

**Reply by hai123vn** [reply]
hi anh đã qua 2 tuần rồi, em vẫn đang hóng bài mới của anh ạ ^^. Chúc anh đầu tuần thật ràng rề năng lượng nhé :3

**Reply by thanhlmm** [reply]
cảm ơn em nhiều, đầu tuần vui vẻ, cuối tuần rồi gì anh nha :)))

**Reply by thanhlmm** [reply]
vao xem bài mới kìa e :)))

**Reply by hai123vn** [reply]
okiee anh ơi mấy nay em bị task dí quá không có lên đây check được nữa @@

**Comment by nkaro13** [comment]
Hi, bạn ui mình chỉ muốn comment rằng mình rất thích blog của bạn, nhất là ở đây bạn nói về công việc chỉ mang lại 2/3 thứ, quan trọng 2 thứ mình muốn là gì. Mình cũng thích đọc các câu chuyện trong các bài blog khác, phần lớn những gì bạn viết là những thứ mình đã nghĩ nhưng chưa bao giờ diễn giải được thành lời. Mong bạn 1 ngày tốt lành và chịu khó viết nhiều nữa nhé.

**Reply by thanhlmm** [reply]
cảm ơn bạn nhiều

**Comment by 1612770** [comment]
Dạo này em rất thích blog của anh, mang lại cho em nhiều góc nhìn và kiến thức mới. Keep it up anh nhé

**Reply by thanhlmm** [reply]
cảm ơn bạn nhiều

**Comment by thanhlmm** [comment]
Mình cũng đã từng zombie như vậy nên nếu ai zombie thì cũng đừng lo nhé ;)

https://thanhle.blog/blog/my-2022

**Reply by thanhlmm** [reply]
![CleanShot 2023-11-28 at 21 54 46@2x](https://github.com/thanhlmm/blog/assets/9281080/9e184747-91d5-4e44-898c-a8fe2fa94a03)

Chà làm việc như 💩 còn bày đặt tự hào

**Comment by huynhthaianhhd** [comment]
hi anh, cảm ơn bài viết này của anh. e thấy mình trong đó zồi ... e thấy mình k còn nhiệt huyết như trước nữa @@ . sẽ tự kiểm điểm bản thân xem mình muốn gì, và mình thiếu gì?. Cảm ơn a nhiều
